java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/logic/impl/UsermanagementDummyImpl.java

```diff
@@ -17,16 +19,41 @@
 @Component
 public class UsermanagementDummyImpl extends AbstractBeanMapperSupport implements Usermanagement {
 
+  @Inject
+  private com.devonfw.application.mtsj.usermanagement.logic.api.Usermanagement usermanagementcore;
```

Comment:
use import to prevent inline FQN please.

Reply:
mhm, there is more odd here. Why do we have two types with the same name:
* com.devonfw.application.mtsj.general.common.api.Usermanagement
* com.devonfw.application.mtsj.usermanagement.logic.api.Usermanagement

This seems to be some kind of design flaw.

Reply:
See 3rd point here:
https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#naming

Reply:
If this was already the case before, you do not need to resolve this within this PR. We can create a new issue for this problem and ignore it for this PR.

Reply:
Yes it was already there, for this i will create an issue to remove mutiple usermanagent

Reply:
issue created - https://github.com/devonfw/my-thai-star/issues/247